src/Illuminate/Database/Console/Migrations/MigrateCommand.php

```diff
@@ -194,7 +194,10 @@ protected function repositoryExists()
     protected function createMissingSqliteDatabase($path)
     {
         if ($this->option('force')) {
-            return touch($path);
+            return (bool) $this->call('db:touch', [
```

Comment:
There is a second `touch()` call on line 212 (of original file), after this commands asks if the user wants it to be created when it is missing.

As you are delegating to the proposed command here, I guess you should delegate there too, right?